.github/workflows/scale-test-100-gce.yaml

```diff
@@ -0,0 +1,222 @@
+        with:
+          project_id: ${{ secrets.GCP_PERF_PROJECT_ID }}
+          version: "405.0.0"
+
+      - name: Clone ClusterLoader2
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: learnitall/perf-tests
+          ref: 67af7e2664b1a5433d098031db0253d5ed4d2a73 # tinker/learnitall/100-node-scale-test
+          persist-credentials: false
+          sparse-checkout: clusterloader2
+          path: perf-tests
+
+      - name: Display version info of installed tools
+        run: |
+          echo "--- go ---"
+          go version
+          echo "--- cilium-cli ---"
+          cilium version --client
+          echo "--- kops ---"
+          ./kops version
+          echo "--- gcloud ---"
+          gcloud version
+
+      - name: Deploy cluster
+        id: deploy-cluster
+        uses: ./.github/actions/scale-tests/create-cluster
+        timeout-minutes: 30
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          control_plane_size: n1-standard-8
+          control_plane_count: 1
+          node_size: e2-medium
+          node_count: 100
+          kops_state: ${{ secrets.GCP_PERF_KOPS_STATE_STORE }}
+          project_id: ${{ secrets.GCP_PERF_PROJECT_ID }}
+
+      - name: Create IG for resource heavy deployments
+        uses: ./.github/actions/scale-tests/create-ig
+        timeout-minutes: 30
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          node_size: e2-standard-8
+          node_count: 1
+          ig_name: heapster
+          kops_state: ${{ secrets.GCP_PERF_KOPS_STATE_STORE }}
+
+      - name: Setup firewall rules
+        uses: ./.github/actions/scale-tests/setup-firewall
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+
+      - name: Install Cilium
+        run: |
+          cilium install ${{ steps.vars.outputs.cilium_install_defaults }}
+
+      - name: Wait for cluster to be ready
+        uses: ./.github/actions/scale-tests/validate-cluster
+        timeout-minutes: 45
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          kops_state: ${{ secrets.GCP_PERF_KOPS_STATE_STORE }}
+
+      - name: Wait for Cilium status to be ready
+        run: |
+          cilium status --wait
+
+      - name: Run CL2
+        id: run-cl2
+        working-directory: ./perf-tests/clusterloader2
+        run: |
+          mkdir ./report
+          export CL2_PROMETHEUS_PVC_ENABLED=false
+          export CL2_ENABLE_PVS=false
+          export CL2_ENABLE_NETWORKPOLICIES=true
+          export CL2_ALLOWED_SLOW_API_CALLS=1
+
+          go run ./cmd/clusterloader.go \
+            -v=4 \
+            --testconfig=./testing/load/config.yaml \
+            --provider=gce \
+            --enable-prometheus-server \
+            --tear-down-prometheus-server=false \
+            --nodes=10 \
```

Comment:
We should set it to 100.

Reply:
Oh nice catch, I missed this, thank you!